src/api/app/components/bs_request_activity_timeline_component.rb

```diff
@@ -26,7 +26,7 @@ def diff_for_ref(comment)
     return unless (ref = comment.diff_ref&.match(/diff_([0-9]+)/))
 
     file_index = ref.captures.first
-    sourcediff = @commented_actions.find(comment.commentable.id).first.webui_sourcediff.first
+    sourcediff = @commented_actions.find(comment.commentable.id).first.webui_sourcediff({ rev: comment.source_rev, orev: comment.target_rev }.compact).first
```

Comment:
Why is this an explicit hash?

Reply:
notice `.compact` at the end, it makes sure that if the value of the rev is nil, it won't be passed along

Reply:
isn't that something you should do on the receiving end?

Reply:
Sure, we can do that